src/lib/query/sriov.rs

```diff
@@ -53,6 +53,8 @@ pub struct VfState {
 #[non_exhaustive]
 pub struct SriovInfo {
     pub vfs: Vec<VfInfo>,
+    pub num_vfs: u32,
+    pub drivers_autoprobe: bool,
```

Comment:
These 2 fields sould be `Option`. Otherwise, if there is an error reading the sysfs files, their values will be `0` and `false`, which is not correct. The correct value is `None`, meaning that we don't know the value.